src/program/raw.rs

```diff
@@ -335,7 +335,8 @@ impl RawProgram {
         let value = unsafe {
             match self.id {
                 Handle::Id(id) => {
-                    assert!(ctxt.version >= &Version(Api::Gl, 2, 0));
+                    assert!(ctxt.version >= &Version(Api::Gl, 2, 0) ||
+                            ctxt.version >= &Version(Api::GlEs, 2, 0));
```

Comment:
It looks like only GLES 3.0 supports GetFragDataLocation. GLES 2.0 should return None.